tron/api/resource.py

```diff
@@ -184,8 +206,9 @@ def getChild(self, run_id, _):
         if run_id in job.action_graph.names:
             action_runs = job.runs.get_action_runs(run_id)
             return ActionRunHistoryResource(action_runs)
-        msg = "Cannot find job run %s for %s"
-        return resource.NoResource(msg % (run_id, job))
+
+        msg = "Cannot find job run '%s' for '%s'"
+        return ErrorResource(msg % (run_id, job))
```

Comment:
Well, idk same with these, I think fstrings will help make the code more readable, but is inconsequential to the user-facing behavior

Reply:
And same here, should've caught it.